### PR TITLE
Adding Defender ATP dependencies doc for clarity

### PR DESCRIPTION
**Pull request by x3nc0n**
added a Defender ATP dependency table for some clarity when enabling on each OS

### CODE REVIEW

File: windows/security/threat-protection/microsoft-defender-atp/deployment-dependencies.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+manager: ayhusain
+audience: ITPro
+ms.collection: 
+- M365-security-compliance
+- m365solution-endpointprotect
+- m365solution-overview  
+ms.topic: article
+---
+
+# Deployment dependancies
```
Comment (illfated) [inline windows/security/threat-protection/microsoft-defender-atp/deployment-dependencies.md:22]: Title typo correction:
```suggestion
# Deployment dependencies
```

File: windows/security/threat-protection/microsoft-defender-atp/deployment-dependencies.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+When deploying Defender for Endpoints, each operating systems may require the Defender ATP agent, app, or the Microsoft Monitoring Agent (MMA). The following table outlines the requirements for each platform:
```
Comment (illfated) [inline windows/security/threat-protection/microsoft-defender-atp/deployment-dependencies.md]: Typo corrections:
```suggestion
# Deployment dependencies
When deploying Defender for Endpoint, each operating system may require the Defender ATP agent, app, or the Microsoft Monitoring Agent (MMA). The following table outlines the requirements for each platform:
```

File: windows/security/threat-protection/microsoft-defender-atp/deployment-dependencies.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+Windows 10 | No | No | [Options](https://docs.microsoft.com/en-us/windows/security/threat-protection/microsoft-defender-atp/onboard-configure#onboarding-tool-options) |
+Windows Server 2008 R2 SP1, 2012 R2, and 2016 | No | Yes | [Options](https://docs.microsoft.com/en-us/windows/security/threat-protection/microsoft-defender-atp/onboard-configure#onboarding-tool-options) |
+Windows Server 1803 and 2019 | No | No | [Options](https://docs.microsoft.com/en-us/windows/security/threat-protection/microsoft-defender-atp/onboard-configure#onboarding-tool-options) |
+macOS | Yes | No | [Options](https://docs.microsoft.com/en-us/windows/security/threat-protection/microsoft-defender-atp/onboard-configure#onboarding-tool-options) |
+Linux Server | Yes | No | [Options](https://docs.microsoft.com/en-us/windows/security/threat-protection/microsoft-defender-atp/onboard-configure#onboarding-tool-options) |
+iOS | Yes | N/A | [App](https://apps.apple.com/us/app/microsoft-defender-atp/id1526737990) |
+Android | Yes | N/A | [Microsoft Endpoint Manager](https://docs.microsoft.com/en-us/mem/endpoint-manager-overview) |
```
Comment (illfated) [inline windows/security/threat-protection/microsoft-defender-atp/deployment-dependencies.md:35]: Codestyle/consistency: add leading MD table cell divider + space:
```suggestion
| Windows 7 SP1 and 8.1 | No | Yes | [Options](https://docs.microsoft.com/en-us/windows/security/threat-protection/microsoft-defender-atp/onboard-configure#onboarding-tool-options) |
| Windows 10 | No | No | [Options](https://docs.microsoft.com/en-us/windows/security/threat-protection/microsoft-defender-atp/onboard-configure#onboarding-tool-options) |
| Windows Server 2008 R2 SP1, 2012 R2, and 2016 | No | Yes | [Options](https://docs.microsoft.com/en-us/windows/security/threat-protection/microsoft-defender-atp/onboard-configure#onboarding-tool-options) |
| Windows Server 1803 and 2019 | No | No | [Options](https://docs.microsoft.com/en-us/windows/security/threat-protection/microsoft-defender-atp/onboard-configure#onboarding-tool-options) |
| macOS | Yes | No | [Options](https://docs.microsoft.com/en-us/windows/security/threat-protection/microsoft-defender-atp/onboard-configure#onboarding-tool-options) |
| Linux Server | Yes | No | [Options](https://docs.microsoft.com/en-us/windows/security/threat-protection/microsoft-defender-atp/onboard-configure#onboarding-tool-options) |
| iOS | Yes | N/A | [App](https://apps.apple.com/us/app/microsoft-defender-atp/id1526737990) |
| Android | Yes | N/A | [Microsoft Endpoint Manager](https://docs.microsoft.com/en-us/mem/endpoint-manager-overview) |
```

File: windows/security/threat-protection/microsoft-defender-atp/deployment-dependencies.md [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+
+
+
```
Comment (illfated) [inline windows/security/threat-protection/microsoft-defender-atp/deployment-dependencies.md:38]: Codestyle: no more than 2 consecutive blank lines needed in regular document spacing.
```suggestion


```

**Review by illfated — APPROVED** [review]
Looks good to me. I was about to say that it might need author verification, but that would create a logic loop error (silly me).

File: windows/security/threat-protection/microsoft-defender-atp/deployment-dependencies.md [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+ms.collection: 
+- M365-security-compliance
+- m365solution-endpointprotect
+- m365solution-overview  
+ms.topic: article
+---
+
+# Deployment dependencies
+## Table of dependencies
+When deploying Defender for Endpoints, each operating system may require the Defender ATP agent, app, the Microsoft Monitoring Agent (MMA) or some combination of them. The following table outlines the requirements for each platform:
```
Comment (adirdidi) [inline windows/security/threat-protection/microsoft-defender-atp/deployment-dependencies.md:24]: 1. "Defender for Endpoint" is the right term and note "Endpoints".
2. "Defender ATP agent" -> "Defender for Endpoint agent".

File: windows/security/threat-protection/microsoft-defender-atp/deployment-dependencies.md [hunk 6]
```diff
@@ -0,0 +1,41 @@
...
+
+|Operating System | Defender Agent/App | MMA Agent | Onboarding Process |
```
Comment (adirdidi) [inline windows/security/threat-protection/microsoft-defender-atp/deployment-dependencies.md:26]: All the links shouldn't contain "/en-us" to enable localization.

File: windows/security/threat-protection/microsoft-defender-atp/deployment-dependencies.md [hunk 7]
```diff
@@ -0,0 +1,41 @@
...
+| macOS | Yes | No | [Options](https://docs.microsoft.com/en-us/windows/security/threat-protection/microsoft-defender-atp/onboard-configure#onboarding-tool-options) |
+| Linux Server | Yes | No | [Options](https://docs.microsoft.com/en-us/windows/security/threat-protection/microsoft-defender-atp/onboard-configure#onboarding-tool-options) |
+| iOS | Yes | N/A | [App](https://apps.apple.com/us/app/microsoft-defender-atp/id1526737990) |
+| Android | Yes | N/A | [Microsoft Endpoint Manager](https://docs.microsoft.com/en-us/mem/endpoint-manager-overview) |
+
+
+## See also
+- [Phase 3: Onboard](onboarding.md)
+- [Plan deployment](deployment-strategy.md)
+- [Defender compatibility](defender-compatibility.md)
```
Comment (adirdidi) [inline windows/security/threat-protection/microsoft-defender-atp/deployment-dependencies.md:41]: This page was deprecated. The right one is microsoft-defender-antivirus-compatibility.md .

**Comment by adirdidi** [comment]
Thank you for the contribution but this page actually feels a bit redundant - and may require constant monitoring to keep it accurate, while the primary pages are kept up-to-date.
The content of this page is covered (in a different form) in those chapters:
https://docs.microsoft.com/en-us/windows/security/threat-protection/microsoft-defender-atp/onboarding
https://docs.microsoft.com/en-us/windows/security/threat-protection/microsoft-defender-atp/onboard-configure

File: windows/security/threat-protection/microsoft-defender-atp/deployment-dependencies.md [hunk 8]
```diff
@@ -0,0 +1,41 @@
+---
+title: Deployment dependencies
+description: Learn about Defender ATP deployment dependencies on each platform OS
```
Comment (adirdidi) [inline windows/security/threat-protection/microsoft-defender-atp/deployment-dependencies.md:3]: "Defender ATP" -> "Defender for Endpoint".

File: windows/security/threat-protection/microsoft-defender-atp/deployment-dependencies.md [hunk 9]
```diff
@@ -0,0 +1,41 @@
...
+| Windows 7 SP1 and 8.1 | No | Yes | [Options](https://docs.microsoft.com/en-us/windows/security/threat-protection/microsoft-defender-atp/onboard-configure#onboarding-tool-options) |
+| Windows 10 | No | No | [Options](https://docs.microsoft.com/en-us/windows/security/threat-protection/microsoft-defender-atp/onboard-configure#onboarding-tool-options) |
+| Windows Server 2008 R2 SP1, 2012 R2, and 2016 | No | Yes | [Options](https://docs.microsoft.com/en-us/windows/security/threat-protection/microsoft-defender-atp/onboard-configure#onboarding-tool-options) |
+| Windows Server 1803 and 2019 | No | No | [Options](https://docs.microsoft.com/en-us/windows/security/threat-protection/microsoft-defender-atp/onboard-configure#onboarding-tool-options) |
+| macOS | Yes | No | [Options](https://docs.microsoft.com/en-us/windows/security/threat-protection/microsoft-defender-atp/onboard-configure#onboarding-tool-options) |
+| Linux Server | Yes | No | [Options](https://docs.microsoft.com/en-us/windows/security/threat-protection/microsoft-defender-atp/onboard-configure#onboarding-tool-options) |
+| iOS | Yes | N/A | [App](https://apps.apple.com/us/app/microsoft-defender-atp/id1526737990) |
```
Comment (adirdidi) [inline windows/security/threat-protection/microsoft-defender-atp/deployment-dependencies.md:34]: The link should be to microsoft-defender-atp-ios.md and not to App Store.

**Comment by x3nc0n** [comment]
> Thank you for the contribution but this page actually feels a bit redundant - and may require constant monitoring to keep it accurate, while the primary pages are kept up-to-date.
> The content of this page is covered (in a different form) in those chapters:
> https://docs.microsoft.com/en-us/windows/security/threat-protection/microsoft-defender-atp/onboarding
> https://docs.microsoft.com/en-us/windows/security/threat-protection/microsoft-defender-atp/onboard-configure

I agree; my goal with this is to document the dependency in one spot for all platforms on agent/app installs and/or MMA. I have a couple of customers confused by what needs to be installed on each platform to enable Defender. Do you think it would be better to add to both of those Onboarding tool options tables in both docs?

**Comment by adirdidi** [comment]
> > Thank you for the contribution but this page actually feels a bit redundant - and may require constant monitoring to keep it accurate, while the primary pages are kept up-to-date.
> > The content of this page is covered (in a different form) in those chapters:
> > https://docs.microsoft.com/en-us/windows/security/threat-protection/microsoft-defender-atp/onboarding
> > https://docs.microsoft.com/en-us/windows/security/threat-protection/microsoft-defender-atp/onboard-configure
> 
> I agree; my goal with this is to document the dependency in one spot for all platforms on agent/app installs and/or MMA. I have a couple of customers confused by what needs to be installed on each platform to enable Defender. Do you think it would be better to add to both of those Onboarding tool options tables in both docs?

Honestly? I don't think we should add this table at all. The first link I included has links to all the different OS types (including the various deployment methods), and those pages have the exact instructions per OS on what to do. I don't see a reason to duplicate the links with another table.
We can take it offline and discuss the customer feedback you got - and see how to improve the existing pages.

File: windows/security/threat-protection/microsoft-defender-atp/deployment-dependencies.md (hunk 5 above)
Comment (illfated) [reply windows/security/threat-protection/microsoft-defender-atp/deployment-dependencies.md:24]: Thanks. I should have checked the document in more detail.

File: windows/security/threat-protection/microsoft-defender-atp/deployment-dependencies.md (hunk 6 above)
Comment (illfated) [reply windows/security/threat-protection/microsoft-defender-atp/deployment-dependencies.md:26]: @x3nc0n: I can add some change suggestions which you can commit as individual changes, unless you prefer to edit the document and make all the manual changes yourself.

**Comment by adirdidi** [comment]
Handling this one offline, closing the PR for now.

**Pull request closed without merging**